ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -48,7 +49,8 @@ type ssh_daemon_options_type = {
     "ClientAliveInterval"               ? long
     "DenyGroups"                        ? string
     "DenyUsers"                         ? string
-    "HostKey"                           ? string
+    "GSSAPIStrictAcceptorCheck"         ? ssh_yesnostring
+    "HostKey"                           ? string[]
```

Comment:
Well, the biggest problem with the current schema is the handling of multi-valued options. The sshd config file has 3 different syntaxes for specifying multiple values:

* space-separated, e.g. `AcceptEnv`, `AllowUsers`, `AuthorizedKeysFile`, `PermitOpen`
* comma-separated, e.g. `AuthenticationMethods`, `Ciphers`
* repeating the same key multiple times, e.g. `HostKey`, `AcceptEnv`, `ListenAddress`.

The current Perl code assumes comma-separated representation if the value is an array, and there's no direct support for the other two syntaxes. Arguably, it would be better to use arrays for multi-line options, and use simple strings (plus string operations on the Pan template side) to handle comma-separated and space-separated values. However, that would break the existing array-typed options - I think there's only one, so it might be possible to hard-code the difference for that specific option in the Perl code.

Latest sshd versions also allow using the "Match ..." syntax on the server-side, to allow overriding the defaults set outside of such `Match` blocks based on various criteria; we don't have any immediate needs to support that, but it would be nice if the schema would eventually allow that as well.